src/drawtypes/ramp.cpp

```diff
@@ -34,11 +34,7 @@ namespace drawtypes {
     vector<icon_t> vec;
     vector<string> icons;
 
-    if (required) {
-      icons = conf.get_list<string>(section, name);
-    } else {
-      icons = conf.get_list<string>(section, name, {});
-    }
+    icons = required ? conf.get_list<string>(section, name) : conf.get_list<string>(section, name, {});
```

Comment:
I think this hurts legibility too much, please revert.

Reply:
@patrick96 sure, will do both :)